config/crater.php

```diff
@@ -16,15 +16,15 @@
         ["code"=>"en", "name" => "English"],
         ["code"=>"fr", "name" => "French"],
         ["code"=>"de", "name" => "German"],
+        ["code"=>"ja", "name" => "Japanese"],
         ["code"=>"it", "name" => "Italian"],
         ["code"=>"lv", "name" => "Latvian"],
         ["code"=>"pt_BR", "name" => "Portuguese (Brazilian)"],
         ["code"=>"sr", "name" => "Serbian Latin"],
+        ["code"=>"ko", "name" => "Korean"],
         ["code"=>"es", "name" => "Spanish"],
         ["code"=>"sv", "name"=> "Svenska"],
-        ["code"=>"sk", "name"=> "Slovak"],
```

Comment:
Hey, @dungphanxuan: I think you've removed the Slovak language from this list. 

We should keep that as it is and add the new languages above.